docs: improve autoscaler guide (#2337)
aiven/terraform-provider-aiven: docs/guides/disk-autoscaler.md

The Console, on other hand can do this:

1. Go to your service page
2. Find the "Service plan usage" section
3. Click on three dots, and "Manage additional storage"
4. Adjust the disk space

## Remove the autoscaler

When you remove the autoscaler integration, the service's disk space will be reset to the default size specified in the service plan. To prevent this automatic resizing:

1. Remove the `aiven_service_integration_endpoint` and `aiven_service_integration` autoscaler resources from your TF files.

    ~> Do not make any changes to the managed service. The `terraform plan` command must show no changes for it.

2. Apply the changes:

    ```bash
    terraform apply
    ```
3. Now the autoscaler doesn't manage the service's disk. The `terraform plan` will show that the `additional_disk_space` will be set to `0B` (default value). Set the value you need and apply your changes.

### Why I can not update the service in the same run?

When the autoscaler exists, the `additional_disk_space` can't be managed by user.

> When it is destroyed during the `apply` command, the field is not set, hence it has no value (zero additional space). Terraform might return an `Provider produced inconsistent final plan` or even worse — trigger the disk resize.

Reviewer: ```suggestion
When it is destroyed during the `apply` command, the field is not set, hence it has no value (zero additional space). Terraform might return a `Provider produced inconsistent final plan` message or trigger the disk resize.
```
